src/framebuffers.h

```diff
@@ -39,7 +39,7 @@ typedef struct {
  *   kart_palette appears to be the palette for all non-wheel elements of the kart (including the driver).
  *   wheel_palette then is the palette for the wheels
  *
- * kart_palette does not appear to be used 100% of the time, more testing is needed to figure out what's going on
+ * kart_palette is "used" in the sense that calling the struct variable directly uses it
```

Comment:
This seems vague and feels like a response to the line you deleted.

I'm not sure if it makes sense by itself.
`* The calling struct variable directly uses the kart_palette` Maybe something along these lines. But this still doesn't make sense to me.

Perhaps something like
`Once loaded, the kart palette is used for the entire race whereas a variety of wheel palettes are chosen as you drive around.`

Perhaps this line is already described below and the line in-question could be deleted?

Reply:
I tried to say that while `kart_palette` is not "called" directly, it is used when you call the struct itself, recall what Tmcg2 said regarding it's usage, i can edit the comment to be more clear regarding that.

Reply:
I read the *whole* comment.

Not super clear. But does okay. Someone can improve it later if they want.